### ISSUE TITLE
[CHORE] - Consistent Error Handling

### ISSUE THREAD

**Issue by naomi-lgbt** [freeCodeCamp/CamperChan-discord#164]
**What needs to be done**

Currently we are inconsistent between logging error messages or sending a message to the end user. My proposed solution would be to ensure both occur during the error events.

For example, if a user calls the `suspend` command without being a moderator, the bot silently ignores them (and logs a message to the console). This makes the bot appear non-responsive or offline - I think we should ensure that the bot responds, even if it is as simple as "You are not authorised to use this command". 

**Additional context**

This could be handled as a separate PR, or tackled with #143 simultaneously.

**Comment by bradtaniguchi** [comment]
Yea I agree, how it currently works is too annoying. 

~~We should wait for #143 to be merged first, since I think there will be a lot of merge conflicts if we don't wait.~~

Updating the log format can be done at any time, but we should wait for #145 (migrate the console.log's to winston calls)

**Comment by naomi-lgbt** [comment]
Sounds good - we could probably tackle it all in one go.

**Comment by naomi-lgbt** [comment]
Once #145 is merged I can do a quick PR to add live responses.
